### DISCUSSION TITLE
Upptime Workflow Runs

### DISCUSSION THREAD

**Discussion by EisenwerkHasenclever** [upptime/upptime#808] [Q&A] [answered]
Somehow the Workflow runs dont seem to work properly.

![image](https://github.com/upptime/upptime/assets/137895844/47b830d2-d020-44ca-a481-df601921dba4)

**Comment by AnandChowdhary** [comment]
Could you please explain a little more about what's going wrong here? From your screenshot I only see green checks, meaning that they succeeded.

**Comment by EisenwerkHasenclever** [comment]
If you have a look at the start time on the right side you see that it isnt running every 5 minutes as it is supposed to do.

Sorry somehow parts of my text got lost while sending.

**Comment by drjoeycadieux** [comment]
Hello,

I would like to know if it  suppose to run every 5 minute to check if website is up, Because when check on my status page last time was around 7pm last night. Thank you

**Comment by AnandChowdhary** [comment, marked as answer]
The reason is mentioned in https://upptime.js.org/blog/2021/01/22/github-actions-schedule-not-working:

> When you set up a GitHub Actions workflow with a schedule, say for once every 10 minutes, you're essentially requesting GitHub to schedule that workflow for you. There is no guarantee that the workflow will run every 10 minutes.
